### DISCUSSION TITLE
IPS output support for sharing a seed with friends?

### DISCUSSION THREAD

**Discussion by murixbob** [TheTrackerCouncil/SMZ3Randomizer#482] [Q&A] [answered]
Hopefully I'm wording this right. Is there any way to have the app give you an option to save the IPS file that would apply the seed, settings, any game patches, etc. so that you can more easily and safely share a seed without having to get the other person to install the app and have to use the same seed and settings string?

Or is this not quite as feasible given the nature of this being a combo rando?

**Comment by MattEqualsCoder** [comment, marked as answer]
Unfortunately, thinking on it for a few, I don't think this is fully feasible due to it being a combo randomizer. If you're unaware, IPS files basically store a diff of the before/after changes to reapply. Because of this, the person would need to run something locally to combine the roms before applying the IPS patch anyway.

**Reply by murixbob** [reply]
Was worth a thought at least. Unless a base combo rom becomes a thing for this, it just wouldn't work. Thanks for the response.
